src/database/DataTypes/Item.js

```diff
@@ -11,19 +13,35 @@ export class Item extends Realm.Object {
   }
 
   get dailyUsage() {
-    return getTotal(this.batches, 'dailyUsage');
+    const endDate = new Date();
+    const startDate = new Date(endDate - USAGE_PERIOD_MILLISECONDS); // 90 Days ago
+    return this.dailyUsageForPeriod(startDate, endDate);
+  }
+
+  // Based on the earliest added ItemBatch associated with this Item
+  // Will return undefined if there are no batches.
+  get addedDate() {
+    if (this.batches.length === 0) return undefined;
+    let itemAddedDate = Date.now();
+    this.batches.forEach(batch => {
+      const batchAddedDate = batch.addedDate;
+      itemAddedDate = batchAddedDate < itemAddedDate ? batchAddedDate : itemAddedDate;
+    });
+    return itemAddedDate;
   }
 
   get earliestExpiringBatch() {
     if (this.batches.length === 0) return null;
-    let earliestBatch = this.batches.find((batch) => batch.totalQuantity > 0);
+    let earliestBatch = this.batches.find(batch => batch.totalQuantity > 0);
```

Comment:
I've run prettier locally which culled the parentheses. We don't have an eslint rule against not parenthesising single params in a arrow function and I'm becoming indifferent towards enforcing this coding style.